commons-analyzer/src/main/scala/com/avsystem/commons/analyzer/BasePackage.scala

```diff
@@ -8,12 +8,23 @@ class BasePackage(g: Global) extends AnalyzerRule(g, "basePackage") {
 
   import global._
 
+  object ImportsOnlyOrEmptyList {
```

Comment:
I believe you can just generalize the pattern match in `validate`, you don't need this extractor